src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php

```diff
@@ -260,7 +260,13 @@ public function createForm($type, $data = null, array $options = array())
      */
     public function createFormBuilder($data = null, array $options = array())
     {
-        return $this->container->get('form.factory')->createBuilder('Symfony\Component\Form\Extension\Core\Type\FormType', $data, $options);
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $type = 'Symfony\Component\Form\Extension\Core\Type\FormType';
+        } else {
+            $type = 'form';
```

Comment:
Please add some @deprecated comment here so that we can spot this easilly on master for removal